Add logs support (#8385)
DataDog/integrations-core: voltdb/README.md

3. [Restart the Agent][4].

#### Log Collection

> _Available for Agent versions >6.0_

Reviewer: ```suggestion
```

I don't think we need this note now

Author: Interesting, thanks. I mostly based this off the new docs spec so it means we may have some changes required there as well. #7891

1. Collecting logs is disabled by default in the Datadog Agent, enable it in your `datadog.yaml` file:

    ```yaml
    logs_enabled: true
    ```

> 2. Add this configuration block to your `voltdb.d/conf.yaml` file to start collecting your RethinkDB logs:

Reviewer 2: ```suggestion
2. Add this configuration block to your `voltdb.d/conf.yaml` file to start collecting your VoltDB logs:
```

    ```yaml
    logs:
      - type: file
        path: "<LOG_FILE_PATH>"
        source: voltdb

>         service: "<SERVICE_NAME>"

Reviewer: ```suggestion
```

    ```

>   Change the `path` and `service` parameter values based on your environment. See the https://github.com/DataDog/integrations-core/blob/master/rethinkdb/datadog_checks/rethinkdb/data/conf.yaml.example for all available configuration options.

Reviewer: ```suggestion
  Change the `path` parameter value based on your environment. See the https://github.com/DataDog/integrations-core/blob/master/rethinkdb/datadog_checks/rethinkdb/data/conf.yaml.example for all available configuration options.
```

Reviewer 2: shouldn't it link to `https://github.com/DataDog/integrations-core/blob/master/voltdb/datadog_checks/voltdb/data/conf.yaml.example`? also, let's add this as a link
